challenges/frontend/challenge.md

```diff
@@ -115,40 +156,68 @@ After the final submission, the interview team will assemble and vote using +1,
-| The submitted code has a clear and modular structure.                                                                | +1                      | -1                         |
-| The candidate outlined the key design points in the design document                                                  | +1                      | -1                         |
-| The code provides examples of tests covering key components                                                          | +1                      | -1                         |
-| The code provides clear error handling and reporting                                                                 | +1                      | -1                         |
-| The app works according to the specifications, no bugs                                                               | +1                      | -1                         |
-| The candidate demonstrates an ability to handle and apply feedback                                                   | +1                      | -1                         |
-| The code is not vulnerable to common web security vulnerabilities                                                    | +1                      | -1                         |
+We want to be as transparent as possible on how we will be scoring your submission. The following table describes areas you will be evaluated on. and how they will affect your overall score.
+
+* The submitted code has a clear and modular structure.
+* The candidate outlined the key design points in the design document.
+* The code provides examples of tests covering key components.
+* The code provides clear error handling and reporting.
+* The app works according to the specifications, no bugs.
+* The candidate demonstrates an ability to handle and apply feedback.
+* The code is not vulnerable to common web security vulnerabilities.
+
+## Questions
+
+It is OK to ask the interview team questions. Some folks stay away from asking questions to avoid appearing less experienced, so we provide examples of questions to ask and questions we expect candidates to figure out on their own.
+
+Here is a great question to ask: 
+
+> Is it OK to pre-generate secret data, like a password hash, and put it in the repository for a proof of concept? I will add a note that we will auto-generate secrets in the future.
+
+It demonstrates that you thought about this problem domain, recognize the trade off and are saving you and the team time by not implementing it.
+
+This is the question we expect candidates to figure out on their own:
+
+> What version of Go should I use? What dependency manager should I use?
+
+Unless specified in the requirements, pick the solution that works best for you.
+
+# Timing
+
+You can split coding over a couple of weekdays or weekends and find time to ask questions and receive feedback.
+
+Once you join the Slack channel, you have between 1 to 2 weeks complete the challenge depending on the challenge you choose.
```

Comment:
Which ones are 1 week and which ones are 2 week?

Reply:
Good point, frontend challenge is only 2 week. I'll fix.